src/desktop/alternatives.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;
```

Comment:
Not sure if it's needed. But in addition to these, in some modules I have also seen a category like `X-SuSE-YaST-System`. @jreidinger Do you have more info about that? I must admit I just do copy&paste for the desktop files. :-)